Kickstarter-iOS/SharedViews/SharedStoryboards/Login.storyboard

```diff
@@ -657,6 +660,9 @@
         </scene>
     </scenes>
     <resources>
+        <namedColor name="backgroundPrimary">
+            <color red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+        </namedColor>
```

Comment:
I'm confused by the changes to this storyboard. Why are some colors changed and others left alone? Also, does the definition above sync with our backgroundPrimary colorset?

Reply:
So, the background set in the storyboard was overwriting the programmatic changes I was making. I couldn't find a way to feature flag colors being set directly in storyboards, and since the Light mode version is the same as what's currently being used across the app, I think this is ok.